net/scripts/gce-provider.sh

```diff
@@ -161,7 +161,11 @@ cloud_CreateInstances() {
   args+=(--image $imageName)
 
   # shellcheck disable=SC2206 # Do not want to quote $machineType as it may contain extra args
-  args+=(--machine-type $machineType)
+  if [[ $machineType == *custom* ]]; then
```

Comment:
How about just punting "--machine-type" to `gce.sh` in all cases.  I feel like I'm gonna get burned by this `if` one day

Reply:
sure, trade-off for the create argument complexity